### PR TITLE
Add basic lobby UI

### PR DESCRIPTION
**Pull request by ben-cook**
image

### CODE REVIEW

File: app/components/index.ts
Comment (Aerilym) [inline app/components/index.ts:1]: I love this, much neater way to sort imports. Lets make sure we do this everywhere

**Review by Aerilym — APPROVED** [review]
Looking good, could it be an issue that we have components in the `app/` directory, considering the `app/` directory is used for page routing? https://beta.nextjs.org/docs/routing/fundamentals

**Comment by Aerilym** [comment]
> Looking good, could it be an issue that we have components in the `app/` directory, considering the `app/` directory is used for page routing? https://beta.nextjs.org/docs/routing/fundamentals

That being said I think [this person on reddit](https://www.reddit.com/r/nextjs/comments/10f9nlf/comment/j4wh5vz/?utm_source=share&utm_medium=web2x&context=3) has good ideas, so it should be fine, and we can have page specific components in the relevant page folder next to the page.tsx file. With generic components going in the components folder

**Comment by ben-cook** [comment]
> > Looking good, could it be an issue that we have components in the `app/` directory, considering the `app/` directory is used for page routing? https://beta.nextjs.org/docs/routing/fundamentals
> 
> That being said I think [this person on reddit](https://www.reddit.com/r/nextjs/comments/10f9nlf/comment/j4wh5vz/?utm_source=share&utm_medium=web2x&context=3) has good ideas, so it should be fine, and we can have page specific components in the relevant page folder next to the page.tsx file. With generic components going in the components folder

This is exactly the correct method to use. I'm a big believer in colocation over """separation of concerns""" which imo has been taken far beyond the logical extreme. I wouldn't mind moving the /components dir out of /app into the top level though, I think that's a sound idea

**Comment by ben-cook** [comment]
I'd also like to point out that colocation has been becoming more popular even without some people realising it. From separate directories for css, html and js... to JSX, which combines the javascript and HTML... to tailwind, which combines JSX with powerful inline styling, effectively back to square one with a single html file with inline scripts and <style> tags lol

**Comment by Aerilym** [comment]
> I'd also like to point out that colocation has been becoming more popular even without some people realising it. From separate directories for css, html and js... to JSX, which combines the javascript and HTML... to tailwind, which combines JSX with powerful inline styling, effectively back to square one with a single html file with inline scripts and <style> tags lol

Sounds good, thanks thats very informative.

**Pull request merged**